src/test/kotlin/com/exactpro/th2/processor/core/event/controller/TestEventController.kt

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright 2022 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.processor.core.event.controller
+
+import com.exactpro.th2.common.grpc.Event
+import com.exactpro.th2.common.grpc.EventBatch
+import com.exactpro.th2.common.grpc.EventID
+import com.exactpro.th2.common.grpc.Message
+import com.exactpro.th2.common.grpc.RawMessage
+import com.exactpro.th2.common.utils.message.toTimestamp
+import com.exactpro.th2.dataprovider.lw.grpc.EventLoadedStatistic
+import com.exactpro.th2.dataprovider.lw.grpc.MessageLoadedStatistic
+import com.exactpro.th2.processor.api.IProcessor
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+import org.mockito.verification.VerificationMode
+import java.time.Duration
+import java.time.Instant
+import java.util.concurrent.TimeUnit
+import kotlin.test.assertFailsWith
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+internal class TestEventController {
+
+    private lateinit var processor: IProcessor
+    private lateinit var eventController: EventController
+
+    @BeforeEach
+    fun beforeEach() {
+        processor = mock {  }
+        eventController = EventController(
+            processor,
+            INTERVAL_EVENT_ID,
+            INTERVAL_START.toTimestamp(),
+            INTERVAL_END.toTimestamp(),
+            BOOK_TO_SCOPES
+        )
+    }
```

Comment:
JUnit by default creates a new class instance per test. So, you can initialize properties right in place